DQMOffline/Trigger/plugins/TagAndProbeBtagTriggerMonitor.cc

```diff
@@ -51,26 +63,44 @@ void TagAndProbeBtagTriggerMonitor::bookHistograms(DQMStore::IBooker     & ibook
   std::string currentFolder = folderName_ ;
   ibooker.setCurrentFolder(currentFolder.c_str());
   
-  pt_jet1_ = ibooker.book1D("pt_jet1","pt_jet1",60,0,600);
-  pt_jet2_ = ibooker.book1D("pt_jet2","pt_jet2",60,0,600);
+  pt_jet1_  = ibooker.book1D("pt_jet1","pt_jet1",jetPtbins_.nbins,jetPtbins_.xmin,jetPtbins_.xmax);
```

Comment:
"necessary" I find it too much, but it is philosophy :P 
I would really encourage you to make use of variable binning, let say